dbt/models/iasworld/schema.yml

```diff
@@ -143,6 +143,19 @@ sources:
               combination_of_columns:
                 - parid
                 - taxyr
+          - column_is_subset_of_external_column:
+              name: pardat_nbhd_matches_spatial_town_nbhd
+              column: nbhd
+              external_model: spatial.neighborhood
+              external_column: town_nbhd
+              additional_select_columns:
+                - parid
+                - taxyr
+              config:
+                # Codes ending in 999 are dummy codes used for some purpose,
+                # although we do not yet know what it is
```

Comment:
I thought about putting this in the docs for the `nbhd` column, but I figure I might as well leave that for our big documentation effort (#26) since I don't have much context on the column yet. But happy to move it if you feel strongly!